config/config.go

```diff
@@ -15,13 +15,14 @@ import (
 
 type Shield struct {
 	// configuration version
-	Version  int                  `yaml:"version"`
-	Proxy    proxy.ServicesConfig `yaml:"proxy"`
-	Log      logger.Config        `yaml:"log"`
-	NewRelic NewRelic             `yaml:"new_relic"`
-	App      server.Config        `yaml:"app"`
-	DB       db.Config            `yaml:"db"`
-	SpiceDB  spicedb.Config       `yaml:"spicedb"`
+	Version           int                  `yaml:"version"`
+	Proxy             proxy.ServicesConfig `yaml:"proxy"`
+	Log               logger.Config        `yaml:"log"`
+	NewRelic          NewRelic             `yaml:"new_relic"`
+	App               server.Config        `yaml:"app"`
+	DB                db.Config            `yaml:"db"`
+	SpiceDB           spicedb.Config       `yaml:"spicedb"`
+	MaxBulkCheckLimit int                  `yaml:"max_bulk_check_limit"`
```

Comment:
Can it be moved to `server.Config` instead?

Reply:
Updated mas @mabdh .